docs/gds-supported-platform/tls-certificates.md

```diff
@@ -0,0 +1,36 @@
+# Public TLS Certificates
+
+TLS certificates can optionally be provisioned via [cert-manager][] in GSP.
```

Comment:
Recommend making more active if it is the user who is doing the provisioning of the TLS certificates. E.g. You can provision TLS certificates using cert-manager